libs/sdk-model/src/insight/sanitization.ts

```diff
@@ -43,6 +40,22 @@ function removeInvalidTotals<T extends IInsightDefinition>(insight: T): T {
     return insightSetBuckets(insight, sanitizedBuckets);
 }
```

Comment:
I just realized this whole function is a bit strange ... sanitize subtotals, but if the totals refer to invalid attributes, only remove those totals. This is unintended, right? Something happened during refactoring or similar

Reply:
You mean the unreachable inner if in
```
 if (totals.length && isSortedOnDifferentThanFirstAttributeInBucket(bucket, sortItems)) {
    bucket.totals = getBucketTotalsWithoutSubtotals(bucket);
    if (totals.length === 0) {
        return bucketSetTotals(bucket, []);
    }
}
```

yes that is strange...

Reply:
I rearranged the function a bit